Tensile/SolutionStructs.py

```diff
@@ -850,10 +850,15 @@ def __init__(self, config):
       else:
         if "DataType" in config:
           self["ComputeDataType"] = DataType(config["DataType"])
+          self["DestDataType"] = DataType(config["DataType"])
         else:
           printExit("NO compute data type, or dest data type, or data type specified")
           self["DataType"] = DataType(0)
 
+    # Modifying ComputeDataType for HHS_BH: if (HHH+HPA), convert it to HHS_BH by setting ComputeDataType to s.
+    if self["ComputeDataType"].isHalf() and DataType(config["DataType"]).isHalf() and self["HighPrecisionAccumulate"]:
```

Comment:
What do we do in the case of BBB+HPA? Do we reject it, or silently change the compute type like this? Since we're updating all the logic files in rocBLAS, I think rejecting might be better. At the very least, I think a warning message would be good.

Reply:
Looks like BBB isn't supported regardless of HPA or not, so modelling the behaviour off of that case isn't an option. I'd be OK with either a printExit or printWarning here depending on what you and/or @zaliu thinks is more appropriate

Reply:
I don't think we reject BBB+HPA this. I guess we treat it the same way (but I am not sure about it).